site/_docs/powered_by.md

```diff
@@ -137,3 +137,7 @@ It uses Calcite to transparently rewrite queries to use materialized views.
 is a SQL interface to a
 <a href="https://tinkerpop.apache.org/">Apache TinkerPop</a>-enabled
 graph database.
+
+### TBDS
+
+[TBDS](https://cloud.tencent.com/product/tbds) (Tencent Big Data Suite) is Big data computing and analysis platform, it uses calcite for query analysis.
```

Comment:
How about ".... is a platform for big data storage and processing. It uses calcite for query analysis."?

Reply:
> How about ".... is a platform for big data storage and processing. It uses calcite for query analysis."?

well ,Let me change it.

Reply:
What does query analysis mean? Parser? or optimizer?

Reply:
> What does query analysis mean? Parser? or optimizer?

SQL parsing and query optimization